models/demos/mamba/tt/mamba_block.py

```diff
@@ -77,6 +76,19 @@ def __init__(self, args: ModelArgs, device, configs, load_fn: Callable):
                 )
             )
 
+        self.use_torch_conv = True
```

Comment:
Probably can just remove this variable and the condition.

Reply:
That was a place holder to select conv version in future.